plugins/lookup/bitwarden.py

```diff
@@ -121,7 +126,11 @@ def _get_matches(self, search_value, search_field, collection_id):
 
         # This includes things that matched in different fields.
         initial_matches = AnsibleJSONDecoder().raw_decode(out)[0]
-
+        if search_field == 'id':
+            if initial_matches == None:
```

Comment:
```suggestion
            if initial_matches is None:
```

Reply:
Thanks for your suggestion! This is fixed in the latest commit.